locksmith/__tests__/controllers/v2/ticketsController.test.ts

```diff
@@ -67,6 +69,29 @@ describe('sign endpoint', () => {
     expect(response.status).toBe(200)
   })
 
+  it('marks ticket as checked-in without auth fails', async () => {
+    expect.assertions(1)
+    const response = await request(app).put(
+      `/v2/api/ticket/${network}/lock/${lockAddress}/key/${tokenId}/check`
+    )
+
+    expect(response.status).toBe(500)
+  })
+
+  it('marks ticket as checked-in fails for non-verifier', async () => {
+    expect.assertions(2)
+    const { loginResponse } = await loginRandomUser(app)
+    expect(loginResponse.status).toBe(200)
+
+    const response = await request(app)
+      .put(
+        `/v2/api/ticket/${network}/lock/${wrongLockAddress}/key/${tokenId}/check`
+      )
+      .set('authorization', `Bearer ${loginResponse.body.accessToken}`)
+
+    expect(response.status).toBe(401)
```

Comment:
It should be 403

Reply:
is not `401` correct? 

user is signed-in but, is not authorized to perform this specific request because is not a verifer

Reply:
nice! changed also the middleware status code to match `403`